tox.ini

```diff
@@ -221,31 +219,28 @@ setenv=
   container_podman: PLAYBOOK = site-docker.yml.sample
 
   rhcs: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_STABLE_RELEASE = luminous
```

Comment:
@leseb will the the next stable release be intended to deploy luminous and/or mimic ?

Reply:
Current master, so next ceph-ansible stable will only deploy starting from Nautilus.